docs/coach.rst

```diff
@@ -18,54 +18,54 @@ View Learner Reports
 
 This is the default view when you select a class from the **Coach** dashboard. It displays the list of learners in the class and the group they are asigned to.
 
-When you click the username of a learner you will see the list of content they accessed and when were they last active on each of the channels:
+When you click the username of a learner, you will see the list of content they accessed and when were they last active on each of the channels:
 
 	.. figure:: img/learner-report-content.png
-	  :alt: Learner reports present quick view of all the content accessed by a learner.
+	  :alt: Learner Reports present a quick view of all the content accessed by a learner.
```

Comment:
I know that you try to avoid American Titles in general, but "Learner reports" is really hard to read for me... I keep reading "reports" as the verb. @radinamatic

Reply:
It's more of a style consistency question, which is known to improve the readability, but I understand what you mean... 🤔